scripts/1M_Glycerol_2nd_pulse.R

```diff
@@ -0,0 +1,25 @@
+###how to analyze only the second pulse when imaging two pulses (i.e. 1M Glycerol experiments) - creates csv file
```

Comment:
I'd suggest renaming this script to exclude '1M_glycerol', if this is intended to be generalizable for olfactory imaging. Generally, the name of the function, eg 'corrected_secondpulse' should be the name of the file.